coderd/workspacebuilds.go

```diff
@@ -428,6 +445,79 @@ func (api *API) postWorkspaceBuilds(rw http.ResponseWriter, r *http.Request) {
 	httpapi.Write(ctx, rw, http.StatusCreated, apiBuild)
 }
 
+func (api *API) notifyWorkspaceUpdated(ctx context.Context, workspaceID uuid.UUID) {
```

Comment:
It seems a bit unfortunate that this method needs to redo a bunch of queries just to get the message across, AFAICT we already have most of the data available in the endpoint calling out to this method.

Not going to enforce a change here, but this could be made more specific for the use-case (for the time-being).

(Additionally, if moved inside `wsbuilder` I believe we would have pretty much all of the data available.)

Reply:
Yeah I totally agree, I've tried to reduce the amount of queries it makes.